priv/templates/nimble_template/.prettierrc.js.eex

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
Since it's an Elixir project, can we use the file format `.prettierrc`, so dropping `.js`. 

It's supported by Prettier:

![image](https://user-images.githubusercontent.com/696529/126604507-7e53bb80-0967-4571-b471-b873a487b577.png)

Instead of using `module.exports` then it could be a JSON or YAML 💡